### DISCUSSION TITLE
Feedback on Potential Scrolling Issue with ScrollMode.OffScreen

### DISCUSSION THREAD

**Discussion by goodgame365** [CoderLine/alphaTab#1754] [Q&A] [answered]
I noticed that when public scrollMode: ScrollMode = ScrollMode.OffScreen;, if the cursor reaches the bottom, scrolling is no longer needed. However, when repeatedly replying, if the starting point exceeds the display range, the screen does not jump to that location.

On the other hand, if public scrollMode: ScrollMode = ScrollMode.Continuous;, this issue does not occur.

I’m not sure if this is a bug.

**Comment by Danielku15** [comment]
Not sure if I understand correctly, maybe something got confused in translation. What do you mean by "when repeatedly replying"? 

Maybe you can setup an example to show what you mean?

**Comment by goodgame365** [comment]
@Danielku15 

when 
public scrollMode: ScrollMode = ScrollMode.OffScreen; like then pic

![未标题-3](https://github.com/user-attachments/assets/05b5aceb-30f3-412e-9f59-1fe6b0e3f9d0)

Please note a start to repeated marking.
When he continues to play after being scrolled off the screen and encounters repeatEnd, the screen does not synchronize and jump to his position


https://github.com/user-attachments/assets/c9922b5e-d273-4172-add4-f28d9dc24331

**Reply by Danielku15** [reply, marked as answer]
Thanks for the details. Looks like a bug to me. 
I entered https://github.com/CoderLine/alphaTab/issues/1766
